web/chat/chat-tabs.react.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+import ChatThreadTab from './chat-thread-tab.react';
+import css from './chat-tabs.css';
+
+type Props = {|
+  children: Array<React.Element<any>>,
+|};
+type State = {|
+  activeTab: string,
+|};
+
+class ChatTabs extends React.PureComponent<Props, State> {
```

Comment:
You call this `ChatThreadTabs` elsewhere. Let's use consistent names for components - the export and import names for component should be the same, and the filename should be the same as the component name

Reply:
my bad, renamed the file and component, but left import names, sorry